spinta/config/__init__.py

```diff
@@ -74,115 +73,106 @@
     'debug': False,
 }
 
-TEMPLATES = {
-    ('backends',): 'default',
-    ('manifests',): 'default',
-}
 
+class Config:
+
+    def __init__(self):
+        self._config = {
+            'cliargs': {},
+            'environ': {},
+            'envfile': {},
+            'default': {},
+        }
+        self.add(CONFIG)
```

Comment:
I do not understand from where this `CONFIG` is coming from? Should it be passed as constructor argument?